### PR TITLE
Fix images and equations display issues.

### PR DESCRIPTION
**Pull request by jamesetouma**
- Remove the <center></center> tags
- Remove <p> </p> tags

@oskooi 
I reverted back to markdown images in all the markdown files and removed \<center>\</cener>  for a few equations. I tested using mkdocs. The images render in both md and html files, however, they are not centered. You can't tell that that is the case with large images but with small ones it's pretty obvious.

### CODE REVIEW

File: doc/docs/css/extra.css [hunk 1]
```diff
@@ -55,3 +55,36 @@ pre code.python {
...
+img[src*='#left'] {
+    float: left;
+}
+img[src*='#right'] {
+    float: right;
+}
+img[src*='#center'] {
+    display: block;
+    margin: auto;
+}
```
Comment (stevengj) [inline doc/docs/css/extra.css:90]: Why do you need both Method 1 and Method 2?

**Comment by jamesetouma** [comment]
I just wanted to see which one you prefer. I like method 2 which is evident
to new editors. I'm fine with either; just let me know which method you'd
like to have in there.

**Comment by stevengj** [comment]
Rebased to fix merge conflicts.

**Comment by stevengj** [comment]
Let's use Method 2.

File: doc/docs/Python_Tutorials/Basics.md [hunk 2]
```diff
@@ -1201,9 +1200,9 @@ from mayavi import mlab
 s = mlab.contour3d(eps_data, colormap="YlGnBu")
 mlab.show()
 ```
-<p align="center">
-  <img src="../images/prism_epsilon.png">
-</p>
+
+![](../images/prism_epsilon.png#center)
```
Comment (oskooi) [inline doc/docs/Python_Tutorials/Basics.md:1204]: With this change, the image reappears in https://meep.readthedocs.io/en/latest/Python_Tutorials/Basics/#visualizing-3d-structures but is *not* centered.

**Comment by jamesetouma** [comment]
yes, I commented on that in a different thread. I'm going to submit another
PR to fix the centering in all the HTML files (readthedocs) but we'll still
have the issue of things not being centered in the MD files. I'll be
looking into this afterwards.

**Pull request merged**